chemprop/cli/train.py

```diff
@@ -758,9 +750,13 @@ def main(args):
         monitor_mode = "min" if model.metrics[0].minimize else "max"
         logger.debug(f"Evaluation metric: '{model.metrics[0].alias}', mode: '{monitor_mode}'")
 
-        tb_logger = TensorBoardLogger(output_dir / "tb_logs")
+        try:
+            trainer_logger = TensorBoardLogger(output_dir, "trainer_logs")
+        except ModuleNotFoundError:
+            trainer_logger = CSVLogger(output_dir, "trainer_logs")
```

Comment:
just call this `logger` to mimic the keyword

Reply:
Unfortunately I can't call it `logger` because that is already taken by the overall logger on line 32 `logger = logging.getLogger(__name__)`.